components/dashboard/src/components/SelectWorkspaceClassComponent.tsx

```diff
@@ -13,6 +13,8 @@ import { PlainMessage } from "@bufbuild/protobuf";
 import { Link } from "react-router-dom";
 import { repositoriesRoutes } from "../repositories/repositories.routes";
 import { useFeatureFlag } from "../data/featureflag-query";
+import { isGitpodIo } from "../utils";
+import { ReactComponent as FullArrow } from "../icons/FullArrow.svg";
```

Comment:
lucide (what we're using on dashboard for icons) has arrow-right https://lucide.dev/icons/arrow-right not sure if it matches our need

Reply:
resolved in [5da1d21](https://github.com/gitpod-io/gitpod/pull/19775/commits/5da1d2138b6a9ca3ca4512e914c55c31b18bdf13)